### PR TITLE
Add translation to all parts

### LINKED ISSUE #893: Mark all strings for translations
Continue where I left from https://github.com/SavandBros/gonevis-dash/pull/878

### PR DESCRIPTION
**Pull request by ArsalanSavand**
Fixes #893

### CODE REVIEW

File: src/public/languages/en.json [hunk 1]
```diff
@@ -24,5 +26,21 @@
...
+  "DELETE": "Delete",
+  "PUBLISHED": "Published",
+  "DRAFT": "Draft",
+  "ACTION": "Action",
+  "REMOVE_SELECTED": "Remove Selected",
+  "PUBLICATION": "Publication",
+  "CREATION": "Creation",
+  "COMMENTS_ENABLED": "Comments enabled",
+  "FEATURED": "Featured",
+  "PASSWORDED": "Passworded",
```
[the remark this answers is not in the document]
Comment (ArsalanSavand) [reply src/public/languages/en.json]: Ok

File: src/public/languages/en.json [hunk 2]
```diff
@@ -24,5 +26,21 @@
...
+  "VIEW_ON_SITE": "View on site",
+  "ADD_TO_NAVIGATION": "Add to navigation",
+  "DELETE": "Delete",
+  "PUBLISHED": "Published",
+  "DRAFT": "Draft",
+  "ACTION": "Action",
+  "REMOVE_SELECTED": "Remove Selected",
+  "PUBLICATION": "Publication",
+  "CREATION": "Creation",
+  "COMMENTS_ENABLED": "Comments enabled",
```
[the remark this answers is not in the document]
Comment (ArsalanSavand) [reply src/public/languages/en.json]: Ok

**Comment by ArsalanSavand** [comment]
@Alir3z4 beside Team and Start page, take a look at site_controller.js file.
Only 3 or 4 lines are not translated at SiteController. Thanks.

File: src/app/account/change_pass/change_pass_controller.js [hunk 3]
```diff
@@ -2,26 +2,31 @@
...
+      });
       return;
     }
+
     // Check if Confirm new password and new password were matched, if so raise an error
     if (form.password !== form.confirm_password) {
-      form.errors = {
-        non_field_errors: ["Confirm password does not match."]
-      };
+      $translate("CHANGE_PASSWORD_ERROR_MATCH").then(function(error) {
```
Comment (Alir3z4) [inline src/app/account/change_pass/change_pass_controller.js]: shouldn't this be `trans.CHANGE_PASSWORD_ERROR_MATCH` ?
or because it's one single string ?

**Comment by Alir3z4** [comment]
@ArsalanSavand I made the `SiteController` complete in 48949994b9f76d611a99a0c035fd16f2868de82f
Please confirm they're working properly, especially the replaceable variables.

Also since https://github.com/SavandBros/gonevis-dash/pull/894/commits/0c20641becc3fefe31bcd761b022e5cd23475745, your latest commit the build is broken.

**Comment by ArsalanSavand** [comment]
I fixed the test.

**Comment by AmirSavand** [comment]
@Alir3z4 @ArsalanSavand I think we need to merge this the sooner the better since other changes are going in and this could have lots of conflicts.

Translation should be done in tiny pull requests or a big pull that is done fast so we don't face conflicts.

Another solution is to pull master into this regularly.

**Comment by Alir3z4** [comment]
Sure, go ahead and merge.
Before merging please update the ChangeLog.

**Comment by Alir3z4** [comment]
This looks good now.
I've fixed some changelog file formatting and now it's looking all good.

You did awesome here @ArsalanSavand .
Keep up the great work.

I'm merging this now.

**Pull request merged**